HACKING.md

```diff
@@ -17,19 +17,30 @@ In the following sections, the section header may indicate whether the
 section applies to the local cluster case (`[LOCAL]`) or the official
 prod case (`[PROD]`).
 
+You'll want to be sure you have KVM available in your cluster.  See
+[this section of the coreos-assembler docs](https://github.com/coreos/coreos-assembler/blob/master/README.md#getting-started---prerequisites).
+
+### [PROD] Using a production OpenShift 4 cluster
+
+This is recommended for production pipelines, and also gives you
+a lot of flexibility.  The coreos-assembler document above has
+multiple options for this.
+
 ### [LOCAL] Set up an OpenShift cluster
 
-First, make sure to install `oci-kvm-hook` on your host system (not in a
-pet container). This is required to ensure that the pipeline has access
-to `/dev/kvm`:
+If you're using `oc cluster up` (which is an older OpenShift with Docker)
+the easiest is to install `oci-kvm-hook` on your host system (not in a
+pet container).  NOTE: The production path for this in modern clusters is
+the [KVM device plugin](https://github.com/kubevirt/kubernetes-device-plugins/blob/master/docs/README.kvm.md)
+linked in the `[PROD]` docs.
 
 ```
 rpm-ostree install oci-kvm-hook # if on OSTree-based system
-dnf install -y oci-kvm-hook # if on traditional
+yum -y install oci-kvm-hook # if on traditional
```

Comment:
Yes, here I am changing a `dnf` to `yum` - *again*.  The reason is quite strong here: People using such an old OpenShift are quite likely also to want to run on CentOS 7, which doesn't speak `dnf`.

Reply:
Yup, makes sense to me.